resources/VERSION

```diff
@@ -0,0 +1 @@
+v0.2.2-59-ga88853a
```

Comment:
> I read the commits individually, because I thought they were independent.

They were, a few rebases ago...

> In the future you could try "fixup" commits and see if that helps.

I'm using it already.

> When a review is done you can do an interactive rebase with the --autosquash flag which will automatically set the correct commits to "squash".

I didn't knew about this one, I'll surely try this.